libmscore/types.h

```diff
@@ -247,6 +247,33 @@ enum class AccidentalType : char {
       ///\}
       };
 
+//---------------------------------------------------------
+//   NoteType
+//---------------------------------------------------------
+
+enum class NoteType {
+      ///.\{
+      NORMAL        = 0,
+      ACCIACCATURA  = 0x1,
+      APPOGGIATURA  = 0x2,       // grace notes
+      GRACE4        = 0x4,
+      GRACE16       = 0x8,
+      GRACE32       = 0x10,
+      GRACE8_AFTER  = 0x20,
+      GRACE16_AFTER = 0x40,
+      GRACE32_AFTER = 0x80,
+      INVALID       = 0xFF
+      ///\}
+};
```

Comment:
This brace should be indented 6 spaces more (like it was in the original code).

Reply:
It is at six spaces.

I followed the same style as the rest of the file.

-Dale

Reply:
>Doesn't it make sense to add noteType property also to Chord type? 

It does make sense. In fact, internally, the Note class delegates the fetch of the property to the parent's chord object.

I'll look at adding that.

-Dale

Reply:
I exposed the noteType property on the Chord object as suggested.
Tested on local build.
-Dale